roles/confluent.variables/defaults/main.yml

```diff
@@ -1,8 +1,14 @@
 ---
 # Custom filters used in this file are defined in filter_plugins/filters.py
 
-confluent_repo_version: 6.1
+confluent_repo_version: "{{ confluent_package_version | regex_replace('^([0-9])\\.([0-9]*).*', '\\1.\\2') }}"
```

Comment:
This is an unrelated change to this PR. I would suggest dropping this change here, and I would also question this change in general. 😉 I think it is nice to see the repo-version in "clear text" - even if semi-duplicated....

Reply:
ya the point is that why would you need to set 
```
confluent_package_version: 6.1.1
confluent_repo_version: 6.1
```
when just `confluent_package_version` is enough. All in all `confluent_package_version` is not in VARIABLES.md and it should be

Reply:
I do not necessarily disagree with the change. 😄 But I think it could be a separate PR, since I prefer PRs with a single concern. And I also think the variable `confluent_repo_version` should be moved to vars (not defaults) - since it is now a deduced variable.

Reply:
ok lets just do 2 prs, thats not that hard